woocommerce/woocommerce
# Fix WooAF changelog typo and release date

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes woocommerce/woocommerce-quality#757.

In my investigation, it seems that the cause of the issue is this: https://github.com/woocommerce/woocommerce/pull/43254#discussion_r1440671263



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a fork of our WooCommerce repository. All the succeeding steps should be done on this fork.
2. Checkout this branch:
   ```bash
   git checkout fix/changelog-date-typo
   ```
3. Comment out the steps and jobs unnecessary for this test -- specifically these steps and jobs: 97985a0ad01c37bf110930a580807776917ef1af. Alternatively, cherry-pick that commit using the command below:
   ```bash
   git cherry-pick 97985a0ad01c37bf110930a580807776917ef1af
   ```
1. Add the `--dry-run true` option in the `Prep accelerated release` step as was done in a41ea994015f6a492b65b6a60cad32b3771d09af. Alternatively, cherry-pick this commit:
   ```bash
   git cherry-pick a41ea994015f6a492b65b6a60cad32b3771d09af
   ```
1. Go to the `Release: Code freeze` workflow page in your fork.
1. Choose a future WooAF version, and take note of its code freeze date and release date. For example, if you chose version `8.6.0.50`, its code freeze date is Feb 8, 2024, and its release date is Feb 13, 2024.
1. Run the workflow using the following inputs:
	- Branch: Select `fix/changelog-date-typo`
	- Time Override: Enter the date-time equal to a few minutes past 12MN of the code freeze date `2024-02-08T00:00:01Z`
	- Skip Slack Ping: Yes
	- Slack Channel Override: Any random text to make sure the Slack notification won't get sent `12345`
1. Wait for the workflow run to finish.
1. Verify the Release calendar in the "Get the  versions for the accelerated and monthly releases" step shows the correct dates.
    <img width="947" alt="Screenshot 2024-01-03 at 6 03 14 PM" src="https://github.com/woocommerce/woocommerce/assets/4509348/ce75c48f-e53e-40c9-bbeb-376fa705fde7">
1. Verify the date argument in the "Prep accelerated release" step correctly shows the expected release date, Feb 13th.
   - It's ok if the "Prep accelerated release" step fails with the `Error: fatal: couldn't find remote ref release/8.6.0.10`. This is because we commented out the step to create the release branch earlier. 
   <img width="1041" alt="Screenshot 2024-01-03 at 6 04 08 PM" src="https://github.com/woocommerce/woocommerce/assets/4509348/0e45c9ec-7645-4f79-99d5-932983518295">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
